protocol-units/settlement/mcr/contracts/script/helpers/deployments.json

```diff
@@ -0,0 +1,14 @@
+{
+    "mainnet": {
```

Comment:
For completeness, can you link to another project that uses this format or else document it yourself?

Reply:
https://github.com/traderjoe-xyz/joe-v2/blob/main/script/config/deployments.json

Reply:
In the comments on the code.